client/src/app/tabs/bpmn/BpmnEditor.js

```diff
@@ -49,7 +49,7 @@ import {
   replaceUsages as replaceNamespaceUsages
 } from '../util/namespace';
 
-import configureModeler from './util/configure';
+import configureModeler from '../common-bpmn/util/configure';
```

Comment:
Maybe we can use something like `bpmn-shared` as a name here? Rationale: we use this syntax in other places, too.

* https://github.com/bpmn-io/dmn-js/tree/develop/packages/dmn-js-shared
* https://github.com/camunda/element-templates-json-schema/tree/master/packages/element-templates-json-schema-shared